pyproject.toml

```diff
@@ -1,9 +1,17 @@
 [build-system]
-requires = ["setuptools",
-            "setuptools_scm>=6.2",
-            "cython==0.29.34",
-            "numpy>=1.25,<2",
-            "extension-helpers"]
+requires = [
+    "setuptools",
+    "setuptools_scm>=6.2",
+    "cython==0.29.34",
```

Comment:
cython 0.29 does not 3.12, need Cython 3.x for that

Reply:
Edit: I was wrong, but a newer 0.29.x release than 34 is required: 0.29.36

Reply:
At least it compiled with 3.0 for me, but I didn't ran all the tests yet

Reply:
3.0 had been in alpha for nearly 3 years, not surprised no one would have touched it before.
Though I think I did use one of the later alphas for a SciPy dev build or something, and did not encounter any serious problems (not sure about performance).

Reply:
Ah, I just saw this. You can ignore my question about Cython over at your PR to my PR. 😸